docs/7_OngoingWork.md

```diff
@@ -0,0 +1,328 @@
+# Transfer learning and fine-tuning
+
+## The objective 
+The objective of this part of the project, is to use a model tht already exist, and to use it's already intelligent structure to use it to do something else.
+
+After the benchmark, we decided that MobileNetv3Large was the most adapted model for our needs. MobileNet is a model that has 1000 classes, so it can recognize 1000 different objects. In our case, we want it to be able to detect people in distress/wounded and people that are safe. So only 2 classes. 
+
+We chose to use leafs to represent people, for there are few datasets available of wounded people.
+
+
+## Fine-tuning with Pytorch
+
+During the last days of the project, we manage to finetune MobileNetv3Large with pytorch. After getting the finetuned model, we managed to make it run on the Raspberry Pi. 
+
+The model had poor results, but the POC is here, and your mission, if you accept it, is to improve and adjust the training.
+
+### Training
+
+First, you will have to download the script files that are in `misc/finetuning/finetuning.py` on the repo on the machine you want to do the training. They might be :  [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ) 
```

Comment:
You says _files_ at plural but you point to a single file.
You should point to `misc/finetuning/`